### PR TITLE
Gem upgrades

### PR DESCRIPTION
**Pull request by huerlisi**
There is a new security release for Rails 3.2

This updates most other gems, some of them still need versioned dependencies. But we're more up to date again, so porting to Rails 4 should be simpler;-)

@luxflux please take a closer look at c88109d as it introduces some changes in the API.

Review/Testing appreciated:-)

### CODE REVIEW

File: spec/api/bookyt/api/bookings_spec.rb [hunk 1]
```diff
@@ -197,7 +197,7 @@
 
     it 'removes the booking' do
       delete "/api/bookings/#{booking.id}", params, headers
-      expect(response.body).to eq('')
+      expect(response.body).to eq('204')
```
Comment (luxflux) [inline spec/api/bookyt/api/bookings_spec.rb]: Hm... HTTP status `204` means `No Content`... So the body should really be `''` :smile: 

Looks like we should add `body false` for these endpoints: https://github.com/ruby-grape/grape#sending-raw-or-no-data

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -200,6 +196,11 @@
   # The default HTTP method used to sign out a resource. Default is :delete.
   config.sign_out_via = :delete
 
+  # ==> Secret
+  # Currently this key will be used on all Bookyt instances. This will be changes when
+  # updating to Rails > 4 where we use it's token infrastructure
+  config.secret_key = 'e3c5de432fa7cd3f80c28a2ec71cc30fc04a7c980e5a2f1ebea620c7eff842905252a7254a67f77db75527abeb0c9d5958ce30e4cf97cc43cb651d68183b9f25'
```
Comment (luxflux) [inline config/initializers/devise.rb]: I am not sure about this one... Can't we handle this like the session secret?

Comment (luxflux) [inline config/initializers/devise.rb]: Lol, just saw that it is handled like the session secret...

**Comment by luxflux** [comment]
@huerlisi I just did a short review. Looking forward to your changes :smile:

File: config/initializers/devise.rb (hunk 2 above)
Comment (huerlisi) [inline config/initializers/devise.rb]: So this is fine with you for now?

File: spec/api/bookyt/api/bookings_spec.rb (hunk 1 above)
Comment (huerlisi) [inline spec/api/bookyt/api/bookings_spec.rb]: Thanks for looking that up. Did change the API endpoints to use `body false`.

File: config/initializers/devise.rb (hunk 2 above)
Comment (luxflux) [inline config/initializers/devise.rb]: Yeah... at least it is consistent bad :P

**Comment by luxflux** [comment]
Looks good from my side...

**Pull request merged**